assets/icons/ic_pen_outlined.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="30" height="30" viewBox="0 0 30 30" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Should create better naming, so when later refactor or changed icon easier, because these icons not using anywhere else, it's better define like these pattern

ic_create_post
ic_create_room
ic_create_event
ic_create_poap
ic_create_collectible